internal/scm/gitea/gitea.go

```diff
@@ -7,6 +7,8 @@ import (
 	"sort"
 	"strings"
 
+	"golang.org/x/exp/slices"
```

Comment:
slices.DeleteFunc now exist in the std slices package now 😄

Reply:
Yeah happy to switch that over. When I coded this up it was before I'd rebased the old branch so I hadn't picked up #402 and thought you were still on go1.19 as a baseline.

FYI scm/github/github.go is also using golang.org/x/exp/maps which is available as just maps now too

Reply:
(I'll update both under this PR)

Reply:
Ah my mistake, looks like [maps.Values](https://pkg.go.dev/golang.org/x/exp/maps#Values) in exp/maps is newer than the stdlib maps — even 1.22 doesn't have it so it probably won't come until go1.23